MapboxNavigation/CarPlayManager.swift

```diff
@@ -195,13 +183,17 @@ public class CarPlayManager: NSObject {
      The most recent search text.
      */
     var recentSearchText: String?
-
+    
+    /**
+     The most recently selected search results.
```

Comment:
nit: documentation doesn't correspond to the property

Reply:
good catch!